docs/content/quickstart/datastream-api-package-guidance.md

```diff
@@ -0,0 +1,242 @@
+# DataStream api package guidance
+
+This guide provides a simple pom example of mysql cdc DataStream api
+
+## frame version
+
+flink 1.13.6  flink mysql cdc 2.4.1
```

Comment:
Flink 1.13 may not be support in the future, you can give an example based on the latest support version.

Reply:
You're right, I plan to update it to Flink 1.18 and CDC 2.5.0 in the future, because at this time, many users who do not use Flink 1.18 will be affected by the Guava version upgrade, just like some users who currently use Flink 1.13 and CDC 2.4.1 will be affected by Guava 30 and Guava 18

Reply:
Could we list all flink version and give the example in Flink 1.17.2 which is the latest stable version Flink CDC supported ? Btw，we can use mysql cdc 2.4.2 version here.